docs/source/guides/speed.rst

```diff
@@ -199,12 +197,25 @@ Read more in our :ref:`accelerators` and :ref:`plugins` guides.
 
 -----------
 
+**************
+Early Stopping
+**************
+
+Usually long training epochs can lead to either overfitting or no major improvements in your metrics due to no limited convergence. Here Early Stopping can help you stop
```

Comment:
```suggestion
Usually, long training epochs can lead to either overfitting or no major improvements in your metrics due to no limited convergence. Using the :class:`~pytorch_lightning.callbacks.early_stopping.EarlyStopping` callback can help you stop
```

Reply:
in the next line, I have already linked it.

Reply:
I would always link them as a habit. Users can click on them at any time.